google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AddAppEngineFlexibleFrameworkSupportToolsMenuAction.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.facet.flexible;
+
+import com.google.cloud.tools.intellij.appengine.facet.AddAppEngineFrameworkSupportAction;
+import com.google.cloud.tools.intellij.appengine.facet.flexible.AppEngineFlexibleSupportProvider.AppEngineFlexibleSupportConfigurable;
+import com.google.cloud.tools.intellij.util.GctBundle;
+import com.intellij.framework.addSupport.FrameworkSupportInModuleConfigurable;
+import com.intellij.framework.addSupport.FrameworkSupportInModuleProvider;
+import com.intellij.openapi.module.Module;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Creates a shortcut in the tools menu to add the App Engine Flexible framework support to a
+ * module.
+ */
+public class AddAppEngineFlexibleFrameworkSupportToolsMenuAction extends AddAppEngineFrameworkSupportAction {
+  public AddAppEngineFlexibleFrameworkSupportToolsMenuAction() {
+    super(GctBundle.message("appengine.flexible.facet.name"),
+        GctBundle.message("appengine.flexible.name.in.message"));
```

Comment:
could you just collapse this into each description? I.e. just have 2 description messages; 1 for flex and 1 for standard. I think that would make things clearer.

Reply:
(and you could get rid of this "name.in.message" message)

Reply:
The name.in.message is used for 2 other dialogs, should I still collapse the description?

Reply:
I ran a 'find usages' and see the "name.in.message" being used only for these new Actions. 1 usage for the flex and 1 usage for the standard one.

Reply:
In AddAppEngineFrameworkSupportAction, the name.in.message is assigned to a FRAMEWORK_NAME_IN_MESSAGE and utilized in the ChooseModule and error dialog

Reply:
Ok. Maybe its the naming thats bothering me. How about:

appengine.flexible.facet.name > appengine.flexible.facet.name.title
appengine.flexible.name.in.message > appengine.flexible.facet.name

Also updating the variable names accordingly.

Otherwise, just running the formatter as I mentioned in another comment and LGTM.